docs/source/extension/extension_migration.rst

```diff
@@ -32,6 +32,21 @@ breaking change when defining fixture values that are array of objects.
 
 See the `Playwright 1.46.0 release notes <https://github.com/microsoft/playwright/releases/tag/v1.46.0>`_ for more information.
 
+
+Shared model
+~~~~~~~~~~~~
+
+The outputs set on the shared cell model are now expected to be wrapped
+in the pycrdt ``Map`` objects rather than provided as plain dictionaries.
```

Comment:
Should we mention `pycrdt` here? It seems that we are talking from the frontend point of view, so maybe we should mention `Y.Map`?
```suggestion
in the ``Y.Map`` objects rather than provided as plain objects.
```

Reply:
Hmm... I think both. But yeas, it makes sense to use `Y.Map` notation as this should be clear that it refers to yjs/pycrdt stack.

Reply:
```suggestion
in the ``Y.Map`` objects rather than provided as plain objects
(or pycrdt ``Map`` objects rather than dictionaries when set on the backend).
```

My point is that there is usage of documents similar to server-side execution in the wild.